src/components/Input.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {ChangeEvent, useState} from "react"
+import {InputProps} from "../interface/InputProps.ts"
+
+const Input: React.FC<InputProps> = ({type, className, maxLength, placeHolder, inputChange}) => {
+    const [inputValue, setInputValue] = useState('')
+
+    const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+        let value = event.target.value
+        setInputValue(value)
+        if (inputChange) {
+            inputChange(value)
+        }
+    }
```

Comment:
r:

- 스코프 내부에서 `value`는 변경되지 않는 변수이기 때문에 const로 선언하는 것이 좀더 안전합니다.
- inputChange처럼 옵셔널 콜백 인자는 옵셔널 체이닝을 통해 간결하게 호출할 수 있어요.

```suggestion
    const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
        const value = event.target.value
        
        setInputValue(value)
        inputChange?.(value)
    }
```

Reply:
수정했습니다.